### PR TITLE
v0.5.0

### PR DESCRIPTION
**Pull request by ouziel-slama**
New features:
- Tower CLI bash auto-completion.
- Auto-synchronize offline hosts and thinclient time with 'router' time.
- Hosts status in taskbar.
- Lock screen and suspend on lid close.
- Power-off button works.
- TowerOS config file (for wallpaper, screen lock timeout and Labwc autostart).
- Added 2 additional TowerS wallpapers to `/var/towercomputers/wallpapers`.
- Enable sound in Thin Client.
- Support one RJ45 cable for the Thin Client.
- `startw` alias.

Documentation:
- Update QA script and usage with new features.
- Improvements and tweaks.

Miscellaneous:
- Use APK to install `tower-cli`.
- Use `doas` instead `sudo`.
- Bugs fixes, code cleaning and tweaks.

### CODE REVIEW

File: tower-build-cli/tower-build [hunk 1]
```diff
@@ -26,18 +25,13 @@ def parse_arguments():
         action='store_true',
         default=False
     )
-    parser.add_argument(
-        '--builds-dir',
-        help="""Directory containing builds necessary to build an image.""",
```
Comment (adamkrellenstein) [inline tower-build-cli/tower-build:31]: What does `"builds"` mean here?

File: tower-lib/towerlib/utils/menu.py [hunk 2]
```diff
@@ -55,39 +55,100 @@ def add_installed_package(host, package):
     # copy desktop files from host to thinclient
     copy_desktop_files(host, package)
 
+STATUS_KEYS = {
+    "name": "Name",
+    "status": "Status",
+    "online-host": "Online Host",
+    "ip": "IP",
+    "toweros-version": "Tower Version",
```
Comment (adamkrellenstein) [inline tower-lib/towerlib/utils/menu.py:63]: "TowerOS Version"

**Review by adamkrellenstein — APPROVED** [review]

**Comment by adamkrellenstein** [comment]
This is going to be a great release! So many nice quality-of-life improvements!

File: tower-build-cli/tower-build (hunk 1 above)
Comment (ouziel-slama) [reply tower-build-cli/tower-build:31]: it's a legacy.. initially the folder contained the NX builds.. now only the TowerOS-Host image. I removed this flag which no longer really makes sense now.

**Pull request merged**